features/admin.actions.v1/components/action-config-form.tsx

```diff
@@ -0,0 +1,841 @@
+                            case AuthenticationType.API_KEY.toString():
+                                setAuthenticationType(AuthenticationType.API_KEY);
+
+                                break;
+                            default:
+                                setAuthenticationType(AuthenticationType.NONE);
+                        }
+
+                        renderAuthenticationPropertyFields();
+                    };
+
+                    return (
+                        <>
+                            <FinalFormField
+                                key="authenticationType"
+                                className="select-field-container"
+                                width={ 16 }
+                                FormControlProps={ {
+                                    margin: "dense"
+                                } }
+                                ariaLabel="username"
+                                required={ true }
+                                data-componentid={ `${ _componentId }-authentication-type-dropdown` }
+                                name="authenticationType"
+                                type={ "dropdown" }
+                                displayEmpty={ true }
+                                label={ t("console:manage.features.actions.fields.authenticationType.label") }
+                                placeholder={ t("console:manage.features.actions.fields." +
+                                    "authenticationType.placeholder") }
+                                component={ SelectFieldAdapter }
+                                maxLength={ 100 }
+                                minLength={ 0 }
+                                options={
+                                    [ ...ActionsConstants.AUTH_TYPES.map(
+                                        (option: AuthenticationTypeDropdownOption) => ({
+                                            text: t(option.text),
+                                            value: option.value.toString() }))
+                                    ]
+                                }
+                                onChange={ handleAuthTypeChange }
+                                disabled={ getFieldDisableStatus() }
+                            />
+                            { renderAuthenticationPropertyFields() }
+                        </>
+                    );
+                };
+
+                return (
+                    <Box
+                        className="box-container"
+                    >
+                        <div className="box-field">
+                            { renderAuthentication() }
+                            { !isCreateFormState && (
+                                <Button
+                                    onClick={ handleAuthenticationChangeCancel }
+                                    variant="outlined"
+                                    size="small"
+                                    className={ "mt-2" }
```

Comment:
do not use these prefixes, use a separate sass file instead. Check other places as well.

Reply:
Fixed with https://github.com/wso2/identity-apps/pull/6722/commits/c514505d74137c56840758c44975197ff519db64